### PR TITLE
Made the Ui uniform by changing the phone number field bg color to white

### LINKED ISSUE #4140: In Patients page the Ui is a bit less uniform some fields have different background colour then the rest.
**Describe the bug**
In Patients page the UI is a bit less uniform some fields have different background colour then the rest.

**To Reproduce**

1. Go to Patients Page

**Expected behavior**
The background color of all input fields should be same in the page.

**Screenshots**
image

### PR DESCRIPTION
**Pull request by patelaryan7751**
## Proposed Changes

- Fixes #4140 
Made the Ui uniform by changing the phone number field bg color to white
image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: src/Components/Common/HelperInputFields.tsx [hunk 1]
```diff
@@ -662,7 +663,9 @@ export const PhoneNumberField = (props: any) => {
       {label && <InputLabel>{label}</InputLabel>}
       <div className="flex items-center">
         <PhoneInput
-          inputClass="bg-gray-200 py-3 text-sm border-gray-200 shadow-none focus:border-primary-400"
+          inputClass={` ${
+            bgColor ? bgColor : " bg-gray-200 "
+          }  py-3 text-sm border-gray-200 shadow-none focus:border-primary-400`}
```
Comment (rithviknishad) [inline src/Components/Common/HelperInputFields.tsx]:
```suggestion
          inputClass={`${bgColor || "bg-gray-200"} py-3 text-sm border-gray-200 shadow-none focus:border-primary-400`}
```

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
@patelaryan7751 was it discussed in EOD and I can see a comment by @rithviknishad

**Comment by patelaryan7751** [comment]
Design For Fields discussed in EOD @nihal467 
image

**Comment by nihal467** [comment]
@patelaryan7751 can you change the existing Screenshot on this PR and replace it with new one

**Comment by patelaryan7751** [comment]
Hey @nihal467 it is the same i have only done some code refactoring it's the whole same thing.

**Comment by nihal467** [comment]
@patelaryan7751 
image

the text-box input area colour in the schema is grey-100, was it not intentionally changed? or my understanding is wrong?

**Comment by patelaryan7751** [comment]
Hey @nihal467 according to the discussion in EOD Gigin told to keep the input box white so i have changed it from grey to white . Gigin told to keep small input box white and textarea fields grey in color

**Comment by patelaryan7751** [comment]
> Design For Fields discussed in EOD @nihal467 image

@nihal467 this is the image he told to refer while having color in input and textarea boxes

**Comment by nihal467** [comment]
test approved

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**